index.html

```diff
@@ -66,179 +72,218 @@ <h2>I'm a Software Developer</h2>
                     ...
                 </ul>
             </div>
-            <img class="bgimage" src="/images/Illustration.png" alt="" />
+
         </section>
-        <section class="myworks">
-            <h2 id="work-headline">My Recent Works</h2>
-            <div class="multi-storybg">
-                <img src="/images/Multi-Post Stories.png" alt="">
+        <section id="myworks">
+            <div class="works-title">
+                <h2 id="work-headline">My Recent Works</h2>
+                <div class="line"></div>
             </div>
-            <h2 class="work-subheading">Multi-Post Stories</h2>
-            <p class="work-specs">A daily selection of privately personalized reads; no accounts or
-                sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown
-                printer took a standard dummy text.</p>
-            <div class="for-flex">
-                <ul>
-                    <li class="flex-items work-items item1">CSS</li>
-                    <li class="flex-items work-items item2">HTM</li>
-                    <li class="flex-items work-items item3">BootStrap</li>
-                    <li class="flex-items work-items item3">Ruby</li>
-
-                </ul>
+            <div class="myworks">
+                <div class="multi-storybg">
+                    <div class="work-image"></div>
+                </div>
+                <div class="div-myworks">
+                    <h2 class="work-subheading">Multi-Post Stories</h2>
+                    <p class="work-specs">A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an
+                        unknown
+                        printer took a standard dummy text.</p>
+                    <ul class="for-flex">
+                        <li class="flex-items work-items item1">CSS</li>
+                        <li class="flex-items work-items item2">HTM</li>
+                        <li class="flex-items work-items item3">BootStrap</li>
+                        <li class="flex-items work-items item3">Ruby</li>
+                    </ul>
+                    <div><button id="myworks-btn" class="work-btn">See Project</button></div>
+                </div>
```

Comment:
- [ ] As requested by the previous code reviewer, It would be better if you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Currently, you have no `margin-top` for this see the project button as it is in the Figma design :blush:
  ### Your design
  ![image](https://user-images.githubusercontent.com/92303891/236616813-cad778fc-b952-4c14-b4e6-5be774b3e959.png)
  
  ### Figma design
  ![image](https://user-images.githubusercontent.com/92303891/236616905-e52d356d-55ba-4fd6-9ad7-61918d54c35f.png)

- [ OPTIONAL ] Why don't you remove the horizontal overflow from your project on both desktop and mobile to show that your project is responsive :+1:

Reply:
I have brought down the "See project" button down a little bit as per the figma design. Thanks a lot, sir again.

I have implemented the overflow-x to hidden in both mobile and desktop version. It removed the horizontal overflow, thanks a lot for pointing it out sir.